libsweep/examples/CMakeLists.txt

```diff
@@ -4,6 +4,9 @@ project(sweep-examples C CXX)
 if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99 -Wall -Wextra -pedantic")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -pedantic")
+elseif(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99 -Weverything")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Weverything")
```

Comment:
Is clang the default compiler for FreeBSD? Does `-Weverything` make sense here or should it be the same flags as above?

Reply:
Yes, Clang is by default shipped with the base system. Actually, you can even change OS X build flags to `-Weverything`, as Xcode installs Clang by default.

FYI, there's [an article](https://embeddedartistry.com/blog/2017/3/7/clang-weverything) explaining what `-Weverything` does.

Reply:
I'm just worried that users will see tons of warnings for minor things, but because FreeBSD folks seem to know what they're doing and what's going on I'm happy to include it :)